data/en/arraygetmetadata.json

```diff
@@ -1,30 +1,37 @@
 {
-	"name":"arrayGetMetadata",
+	"name":"getMetadata",
```

Comment:
@srbaxter looks like you are overwriting the docs for `arrayGetMetadata` with docs for `getMetadata` 

`arrayGetMetadata` is a valid function, I think you had meant to modify `getmetadata.json` instead? 

Changes to the other files look good.

Reply:
Whoops, yes, sorry